src/en/proposals/dragonjspider-Wawaspecies.md

```diff
@@ -0,0 +1,64 @@
+
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/ae7437cf-f4d3-42c1-8798-b8ce24a2725f)
+![image](https://github.com/Dragonjspider/Wawa-Player-Species-/assets/140023808/69d00309-faae-41e0-a6aa-9e2916052de3)
+
+
+
+## Species
+Our Official name for the species would be Wawa (proposed by fairlysadpanda). Whilst this isn't exactly "creative" for them, we believe a distinct but still related name suits them. We're not going to be naming them Scug(s), or Slugcat(s), because of more specific issues with copyright and such, and lack of explicit written permission.
+For personalization they will be able to have most colors for skin, similar to lizards or moths.
+Anything unmentioned will be similar to humans, like having two hands or a backpack slot.
+
+The conventional naming scheme for them, while mostly a suggestion, would be as follows: Number, Object. E.G: Seven-Hundred Nukies, Twenty-Eight Wounds, Seventy-Three Bombs. (suggested by kiterny)
+Alternatively, they could use a single word describing their character, similar to how players name lizards do, but I felt this might make them too similar in character to lizards. 
+Or finally, it could be pet names, which may be too far off what would normally be acceptable, but may be considered for names. (because of course people would name these things silly names and stuff)
+
+## Abilities
+- They will deal additional piercing damage with spears (+4 damage with spears) (suggested by thecatbastard, cornerdogs, dragonjspider)The following is the time it would take to crit a human with no resists: 
+  - 1h spear: 12 > 16 : Crits in 9 vs 7 
+  - 2h spear: 16 > 20 : Crits in 7 vs 5 
+  - 1h Reinforced spear: 15 > 19 : Crits in 7 vs 6
+  - 2h Reinforced spear: 19 > 23 : Crits in 6 vs 5 
+  - 1h Plasma spear: 18 > 22 : Crits in 6 vs 5
+  - 2h Plasma spear: 22 > 26 : Crits in 5 vs 4 
+  - 1h Uranium spear: 18 > 22 : Crits in 6 vs 5
+  - 2h Uranium spear: 22 > 26 : Crits in 5 vs 4
+
+
+* They will be small in size, similar to dwarfs, making them easier to shove down than other species.
+* They will take 20% additional damage from pierce, making Scug vs Scug combat even more lethal, and encourages a nimble playstyle rather than running face first into a gunfight.
+* They will have 10% resist to either blunt or slash, undecided. (suggested by piksqu, vytian, thecatbastard, dragonjspider)
+* They will be able to run under tables freely, but not under doors, windoors, or plastic flaps. (suggested by thecatbastard, piksqu, vytian)
+
+
+## Optional/experimental abilities
+This section will go over ideas that may be too strong, difficult to code, or other various reasons.
+
+* Once dead, they will have a 30 minute respawn time, respawning at the last bed they slept in. (suggested by erisws)
+* Maybe during crit, they could have a very small amount of asphyx, but slowly regenerate brute naturally. (suggested by vytian) Something like "reverse slimes"
```

Comment:
I don't get exactly what you mean? They asphyxiate slower but heal brute in crit? As long as the asphyxiation is higher than their passive healing (so that they'll never heal from crit, just "convert" their damage into airloss), I'd be fine trying it anyways.

Reply:
I'd have to ask vytian to clarify this, I like the idea, since they'd have a tough time in combat, it would be nice to have something to balance that out.